pkg/engine/query/source.go

```diff
@@ -20,6 +22,43 @@ type FilesystemSource struct {
 	Source string
 }
 
+var (
+	_, b, _, _ = runtime.Caller(0)
+	basepath   = filepath.Dir(b)
+)
+
+// GetGenericQuery returns the library.rego for the platform passed in the argument
+func getPathToLibrary(platform string) string {
+	var basePath = "../../../assets/queries/generic/"
+	var genericPath = filepath.Join(basepath, basePath)
```

Comment:
There's an issue here that is causing no queries to be loaded in integration tests.